homeassistant/components/wallbox/number.py

```diff
@@ -45,6 +50,9 @@ async def async_setup_entry(
     coordinator: WallboxCoordinator = hass.data[DOMAIN][entry.entry_id]
     # Check if the user is authorized to change current, if so, add number component:
     try:
+        await coordinator.async_set_energy_price(
```

Comment:
This shouldn't be here -> number component is only loaded if you are admin (you can change the current is the check), don't make it dependend on price changes